### PR TITLE
Bugfixes

### LINKED ISSUE #9906: Deep fried clown goblins return mask and shoes; for infinite, deep fried goblins
Clown goblins return the mask and shoes after being deep fried, as oppossed to just disappearing and leaving the deep fried corpse behind. This does allow for infinite food in the form of deep fried goblin products, as the item is accepted by the candy machine and cereal maker, (couldn't try the oven and ice cream machine yet).

...I wouldn't mind this bug be left alone for a while, though

### LINKED ISSUE #9954: Gun sprites no longer reliably indicate charge level when EMP'd
After being EMP'd, they seem to indicate that they have a charge still (an orange or partial red/blue indicator), and yet they do not.

### LINKED ISSUE #10284: Tracking Beacon
(WEB REPORT BY: siliconeagle REMOTE: 172.93.109.202:7777)
Frequency change on the tracking beacon does not appear to work. When combined with the locator device from xenoarch it cannot properly be targeted.

### PR DESCRIPTION
**Pull request by clusterfack**
Fix tracking beacon frequency setting
It was calling format frequency which converts the number to num/10, then calling sanitize frequency which clamps it between 1200 and 1600, when in fact it needs to be the other way around.
Fixes #10284

Makes gun sprites update correctly on EMP
Makes power cells remove a percentage of max charge instead of a hardcoded value
Fixes #9954 

Fixes #9906

### CODE REVIEW

File: code/modules/power/cell.dm [hunk 1]
```diff
@@ -118,9 +118,7 @@
 		rigged = 1 //broken batterys are dangerous
 
 /obj/item/weapon/cell/emp_act(severity)
-	charge -= 1000 / severity
-	if (charge < 0)
-		charge = 0
+	charge = max(charge - maxcharge/severity, 0)
```
Comment (Kurfursten) [inline code/modules/power/cell.dm]: I feel like everything about this line is wrong. Charge-maxcharge will always give a negative number, and high severities are weaker (3 is edge, 1 is middle) iirc.

So basically since this will always be a negative number or zero, it will max to 0.

Comment (clusterfack) [inline code/modules/power/cell.dm]: Well its good that its not (charge - maxcharge)/severity then, but charge - (maxcharge/severity)

Comment (Kurfursten) [reply code/modules/power/cell.dm]: Does it make sense that if you have two power cells with 1000 charge left, a normal cell loses less charge than a hyper cell?

That seems unintuitive. On the other hand, this is potentially a huge nerf to borgs so that's interesting.

Comment (clusterfack) [inline code/modules/power/cell.dm]: Trying it out it does seem a tad unbalanced, choosing a way that scales appropriately from 1,000 charge all the way up to 30,000 charge is somewhat challenging.

Comment (Kurfursten) [reply code/modules/power/cell.dm]: The big impact of this is of course that, if you have a maxed out hyper cell and get hit from a direct hit of an ion rifle, it should zero out your battery. So this is a big buff to ion rifles, borgs really finally have a reason to fear them. Knocking out APCs in one hit too, so potentially big affects on malf as well. Since the 2sev area goes through windows, you only have to thermite into the core and shoot one of the windows twice. If my understanding of ion rifle severities is correct, of course.

[Balance] tag?

Comment (clusterfack) [inline code/modules/power/cell.dm]: Yes that is how the CURRENT formula works, I'm looking at ways to change it so it does about 25-50% for 1/2 at 1000, and about 10-20% at 1/2 on a hyper cell.

File: code/modules/power/cell.dm [hunk 2]
```diff
@@ -118,9 +118,8 @@
 		rigged = 1 //broken batterys are dangerous
 
 /obj/item/weapon/cell/emp_act(severity)
-	charge -= 1000 / severity
-	if (charge < 0)
-		charge = 0
+	var/powerloss = round(16 * sqrt(maxcharge) / severity, 50) //at severity 1, ~500 for 1000 power cells, ~2750 for 30,000 power cells
```
Comment (clusterfack) [inline code/modules/power/cell.dm]: At most effective severity (1)
Drains about 50% of a 1000 power cell, 
Drains about 20% of a 7500 power cell,
Drains about 10% of a 30,000 power cell.

Comment (Kurfursten) [reply code/modules/power/cell.dm:121]: 32% of a 2500 cell, iirc APCs use these

File: code/game/response_team.dm [hunk 3]
```diff
@@ -76,7 +76,7 @@ client/verb/JoinResponseTeam()
 
 			ticker.mode.ert += new_commando.mind
 
-			message_admins("[new_commando]/[usr.key] has joined the Emergency Response Team.")
+			message_admins("[new_commando]/[key_name(usr)] has joined the Emergency Response Team.")
```
Comment (ComicIronic) [inline code/game/response_team.dm]: Should be key_name(new_commando), that way you can take out the first bit.

**Comment by clusterfack** [comment]
Okay I made that change

**Comment by Kurfursten** [comment]
>Balance PR
>No changelog

:confounded:
